src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
```

Comment:
Since Random inline policy is meant as a stress tool, would it make sense to add a new stress mode

STRESS_INLINE_RANDOM?

See compiler.h by searching STRESS_MODES to see the list of existing stress modes. 

In such a case a jistress=1/2 setting would trigger random inline policy for some of the methods. If one wants random policy for all methods we could use

complus_jitstress=1
complus_jitstress_ModeNames=STRESS_INLINE_RANDOM